surveillance/sentry.go

```diff
@@ -66,7 +66,9 @@ var (
 // Handles an error by capturing it on Sentry and logging the same on STDOUT
 func (wrapper *Sentry) Capture(err error, _panic bool) {
 	if err != nil {
-		if wrapper.client != nil {
+		// Do not log to sentry if the error is ignorable.
```

Comment:
Will this be better placed in the `utils` package? Sorry I forgot that the utils package in vcore instead of fsm.

Reply:
The idea is to at least log the error in case it isn't being captured in Sentry so that anyone trying to figure out what went wrong can have a look at the application logs while developing/testing something.